00-get-started/02-ingest-pg-cdc.md

```diff
@@ -49,10 +65,23 @@ CREATE USER rw WITH PASSWORD 'abc123' REPLICATION LOGIN CREATEDB;
 Next, grant the user with following privileges so we can access the desired PostgreSQL data from RisingWave. Fill in the database name accordingly based on where your data is located.
 
 ```sql
-GRANT CONNECT ON DATABASE <database_name> TO <username>;   
-GRANT USAGE ON SCHEMA <schema_name> TO <username>;  
-GRANT SELECT ON ALL TABLES IN SCHEMA <schema_name> TO <username>; 
-GRANT CREATE ON DATABASE <database_name> TO <username>;
```

Comment:
This `GRANT CREATE` only grants the user to create schemas in this database. Check the [official PostgreSQL documentation](https://www.postgresql.org/docs/9.1/sql-grant.html).

```
CREATE
For databases, allows new schemas to be created within the database.

For schemas, allows new objects to be created within the schema. To rename an existing object, you must own the object and have this privilege for the containing schema.

For tablespaces, allows tables, indexes, and temporary files to be created within the tablespace, and allows databases to be created that have the tablespace as their default tablespace. (Note that revoking this privilege will not alter the placement of existing objects.)
```